pkgs/applications/office/csv2odf/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib, python3, fetchurl }:
+
+let
+  inherit (python3.pkgs) buildPythonApplication;
+in
```

Comment:
this is no longer needed, otherwise LGTM

Reply:
@jonringer: Removed it.